File: Elixir/start-code/startcode.exs
# Willkommen zu meinem Elixir-Programm

IO.puts "Willkommen zu meinem Elixir-Programm!"

IO.write "Bitte geben Sie Ihren Namen ein: "
name = IO.gets("") |> String.trim()  # Liest die Eingabe des Nutzers

IO.puts "Hallo, #{name}! Schön, dass Sie hier sind."
IO.puts "Das Programm wurde beendet."

File: Dart/start-code/startcode.dart
import 'dart:io';

void main() {
  // Begrüßung
  print('Willkommen zu meinem Dart-Programm!');

  // Nutzereingabe
  stdout.write('Bitte geben Sie Ihren Namen ein: ');
  String name = stdin.readLineSync() ?? ''; // Liest die Eingabe des Nutzers

  // Begrüßung mit dem Namen
  print('Hallo, $name! Schön, dass Sie hier sind.');

  // Programm beenden
  print('Das Programm wurde beendet.');
}
